sources/europe/lu/Luxembourg-GeoportailOrtho2017.geojson

```diff
@@ -0,0 +1,931 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "url": "https://data.public.lu/en/datasets/bd-l-ortho-webservices-wms-et-wmts/",
+            "text": "Administration du Cadastre et de la Topographie",
+            "required": false
+        },
+        "name": "Ortho 2017 geoportail.lu",
+        "end_date": "2017-06-22",
+        "url": "https://{switch:wmts3,wmts4}.geoportail.lu/opendata/wmts/ortho_2016/GLOBAL_WEBMERCATOR_4_V3/{zoom}/{x}/{y}.jpeg",
+        "max_zoom": 20,
+        "start_date": "2017-06-14",
+        "license_url": "https://data.public.lu/en/datasets/bd-l-ortho-webservices-wms-et-wmts/",
+        "country_code": "LU",
+        "type": "tms",
+        "id": "lu.geoportail.opendata.ortho2016",
```

Comment:
should be …ortho2017, right?

Reply:
Indeed, well spotted! The URL too, actually.